src/pages/verify/guides/telemetry.mdx

```diff
@@ -0,0 +1,46 @@
+---
+product: verify
+category: Guides & Tools
+sort: 10
+title: Telemetry & observability
+---
+
+Criipto offers telemetry and observability into authentication flows based on OpenTelemetry.
+
+## Disclaimer
+
+Criipto may at any time change the contents of the traces or the formats of headers/query parameters.
```

Comment:
Should this be in one of the warning/alert boxes instead?

Reply:
aac4363